tests/typ/autocomplete/showcase.typ

```diff
@@ -0,0 +1,15 @@
+// Autocomplete: true
+// Ref: false
+---
+// Autocomplete contains: -1 "int", "if conditional"
+// Autocomplete excludes: -1 "foo"
+#i
+
+--- 
+
+// Autocomplete contains: -1 "len", "first", "last", "at", "push", "pop", "insert", "remove", "slice", "contains", "find",
+// Autocomplete contains: -1 "position", "range", "filter", "map", "enumerate", "zip", "fold", "sum", "product", "any",
+// Autocomplete contains: -1 "all", "flatten", "rev", "split", "join", "intersperse", "sorted", "dedup", "first", "last", "at", "pop", "push",
+// Autocomplete contains: -1 "insert", "remove",
```

Comment:
I don't think we need to list all methods here, that's the idea of the contains after all